packages/mdc-slider/foundation.ts

```diff
@@ -463,9 +463,9 @@ export class MDCSliderFoundation extends MDCFoundation<MDCSliderAdapter> {
     if (this.step_ && !valueSetToBoundary) {
       value = this.quantize_(value);
     }
-    if (value < min) {
+    if (value <= min) {
```

Comment:
Can you please add / update unit tests?

Thanks!

Reply:
Thanks for the reply.
Ok, I added two tests which simulate the bug.

While I was looking into it I found that seems like the bug is caused by `toLocaleString` function call in next line:
https://github.com/material-components/material-components-web/blob/cefcad79644350f05bc1c92efc07f9f1349d0f97/packages/mdc-slider/component.ts#L120
Here is simple demo: https://codepen.io/Tester456/pen/JjPmPmo
So maybe this bug can be fixed there instead of where I tried to fix it initially.

Reply:
Hmm, seems like original fix is only fixing edge cases when "0" value is on the left or right side of the slider, but it doesn't fix the bug when "0" value is located in the middle:
added 3rd slider https://codepen.io/Tester456/pen/oNvawqw

So maybe fixing the bug in the mentioned above line will be a good idea.
Also because of this, probably two tests can be replaced with just one which checks value "0" in the middle.

Reply:
Replaced two tests with only one and changed the fix.